### ISSUE TITLE
typo in example code

### ISSUE THREAD

**Issue by DonBower** [adafruit/Adafruit_CircuitPython_SCD30#8]
This line fails:
```
print("CO2:", scd.CO2, "PPM")
```

it should read:
```
print("CO2:", scd.eCO2, "PPM")
```

**Comment by tannewt** [comment]
Hi @DonBower, the field was just changed to CO2 because it's an actual measurement of CO2 instead of estimated. Please make sure you have the latest library version to match the example code.

**Comment by DonBower** [comment]
Interesting.
I ran the `sudo pip3 install adafruit-circuitpython-scd30` command about less than an hour before I made the comment.

**Comment by tannewt** [comment]
@DonBower Did you get the example from here? We may have updated here without releasing. (I released 2.0.0 right after replying to you yesterday.)

**Comment by DonBower** [comment]
yea, once i did a pip3 --upgrade, it came back with .CO2, not .eCO2.  I think we just must have passed in the mail.
